extras/gomock.bzl

```
@@ -206,6 +206,7 @@ def _gomock_reflect(name, library, out, mockgen_tool, **kwargs):
     mockgen_model_lib = _MOCKGEN_MODEL_LIB
     if kwargs.get("mockgen_model_library", None):
         mockgen_model_lib = kwargs["mockgen_model_library"]
+        kwargs.pop("mockgen_model_library")
```

```suggestion
    mockgen_model_lib = kwargs.pop("mockgen_model_library", _MOCKGEN_MODEL_LIB)
```
should be equivalent, but please test it.